src/edu/csus/ecs/pc2/core/PacketHandler.java

```diff
@@ -3642,6 +3640,20 @@ public void loadSettingsFromRemoteServer(ContestLoader loader, Packet packet, Co
 
         loader.addRemoteLoginsToModel(contest, controller, packet, remoteSiteNumber);
         
+        loader.addJudgementsToModel(contest, controller, packet);
+
+        if (isServer()) {
+            try {
+                contest.storeConfiguration(controller.getLog());
```

Comment:
I don't understand why the process of fixing bug 292 (which is about judgements not being added to the model) involves adding a new call to `storeConfiguration()`.   Is there some reason this is within the scope of bug 292?

Reply:
Yes.   The judgements and other settings (from other sites) were not being saved to disk.